src/codegen/tests/parameters_tests.rs

```diff
@@ -34,42 +31,6 @@ fn function_all_parameters_assigned() {
     insta::assert_snapshot!(result);
 }
 
-#[test]
-fn function_empty_input_assignment() {
```

Comment:
I'm assuming this got covered elsewhere?

Reply:
With these changes, this case no longer results in an error/diagnostic. I think I meant to change this to generate a snapshot instead of checking diagnostics as I did for the program_empty_inout_assignment test. Will change this accordingly.